game-core/src/main/java/games/strategy/triplea/ui/KeyBindingSupplier.java

```diff
@@ -47,9 +47,10 @@ static KeyStroke fromKeyEventCode(final int code) {
   }
 
   static KeyStroke fromKeyEventCode(final int code, final ModifierKey modifierKey) {
-    Preconditions.checkArgument(
-        !KeyEvent.getKeyText(code).toUpperCase().contains("UNKNOWN"),
-        "Be sure to use a constant from 'KeyEvent', unknown key constant: " + code);
-    return KeyStroke.getKeyStroke(code, modifierKey.modifierMask);
+    final KeyStroke result = KeyStroke.getKeyStroke(code, modifierKey.modifierMask);
+    if (result.toString().contains("UNKNOWN")) {
```

Comment:
Is `toUpperCase` not needed?

Reply:
Nope, the `UNKNOWN` in `toString()` is all caps already

Reply:
Okay, I think I put that in there in case it changes in the future or is system dependent.